packages/rocketchat-livestream/client/views/liveStreamTab.html

```diff
@@ -0,0 +1,28 @@
+<template name="liveStreamTab">
+	<div class="contextual-bar__header flex-tab__header">
+		<div class="title">
+			<h2>{{_ "Livestream_title" }}</h2>
+		</div>
+	</div>
+	<div class="contextual-bar__content flex-tab-bar">
+		{{#if canEdit}}
+			{{#if editing}}
+				<form class="liveStreamTab_form">
+					<input type="text" name="streamingOptions" class="content-background-color editing" placeholder="{{_ "Livestream_url"}}" value="{{ streamingSource }}"/>
+					<button type="button" class="button js-cancel" title="{{_ 'Cancel'}}" >{{_ "Cancel"}}</button>
+					<button type="button" class="button primary js-save">{{_ "Save"}}</button>
+				</form>
+			{{else}}
+				<div class="setting-block">
```

Comment:
use __ and use `setting__block` to set margin-bottom on each field (input/buttons/iframe)